src/services/anchor-service.ts

```diff
@@ -81,6 +80,11 @@ const logAnchorSummary = async (
     results
   )
 
+  Metrics.recordObjectFields('anchorBatch', anchorSummary)
+  Metrics.recordRatio('anchorBatch_failureRatio',
```

Comment:
@stbrody, Golda and I discussed this ratio as the right thing to capture for your request. Does this seem reasonable?

Golda's also recording more fields from the summary, which will probably be handy  to visualize trends for.

Reply:
yeah I think that seems right, thanks!